engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala

```diff
@@ -29,12 +29,12 @@ class AvroUtils(schemaRegistryProvider: SchemaRegistryProvider[_]) extends Seria
 
   def latestKeySchema(topic: String): Schema =
     handleClientResponse {
-      schemaRegistryClient.getLatestSchema(AvroUtils.keySubject(topic))
+      schemaRegistryClient.getLatestFreshSchema(AvroUtils.keySubject(topic))
```

Comment:
This shouldn't be used in utils - they are invoked in engine and it is the only place where you should use cached version. Instead of this it should be used in `SchemaRegistryKafkaAvroProvider.typeDefinition`